Sources/AsyncHTTPClient/HTTPHandler.swift

```diff
@@ -604,6 +606,12 @@ internal class TaskHandler<T: HTTPClientResponseDelegate>: ChannelInboundHandler
                 /// Some HTTP Servers can 'forget' to respond with CloseNotify when client is closing connection,
                 /// this could lead to incomplete SSL shutdown. But since request is already processed, we can ignore this error.
                 break
+            case .head, .body:
+                if self.ignoreNIOSSLUncleanShutdownError {
+                    /// we can also ignore this error like .end .
+                    break
+                }
+                fallthrough
```

Comment:
Can I propose using a `where` clause here? That would be:

```swift
case .head where self.ignoreNIOSSLUncleanShutdownError,
     .body where self.ignoreNIOSSLUncleanShutdownError:
    break
```

That avoids the need for a fall through or the internal `if`.

Reply:
Yes, it's better to use internal if. it was updated.